grails-app/services/org/broadinstitute/orsp/CrowdService.groovy

```diff
@@ -1,131 +1,118 @@
 package org.broadinstitute.orsp
 
-import grails.async.PromiseList
 import groovy.util.logging.Slf4j
-import groovyx.net.http.Method
-import groovyx.net.http.HTTPBuilder
-import org.broadinstitute.orsp.config.AppInfoConfiguration
 import org.broadinstitute.orsp.config.CrowdConfiguration
+import com.google.cloud.bigquery.BigQuery
+import com.google.cloud.bigquery.BigQueryOptions
+import com.google.cloud.bigquery.FieldValueList
+import com.google.cloud.bigquery.Job
+import com.google.cloud.bigquery.JobId
+import com.google.cloud.bigquery.JobInfo
+import com.google.cloud.bigquery.QueryJobConfiguration
+import com.google.cloud.bigquery.TableResult
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.ServiceAccountCredentials;
```

Comment:
All changes to this class should be reverted and put into a separate PR that just addresses BTRX-600. There should be no code that calls `CrowdService` since that is handled via a cronjob on the current Broad VM.

Reply:
done